src/cmd-init

```diff
@@ -169,6 +169,7 @@ mkdir -p src
             if [ -n "${COMMIT}" ]; then
                 git -C ./config fetch origin "$COMMIT"
                 git -C ./config reset --hard "$COMMIT"
+                git -C ./config submodule update
```

Comment:
I usually pass `--init` because it doesn't hurt, and I also think `--recursive` is a best practice now.